pkg/didcomm/common/model/ack.go

```diff
@@ -22,14 +22,14 @@ type Ack struct {
 	ID          string            `json:"@id,omitempty"`
 	Status      string            `json:"status,omitempty"`
 	Thread      *decorator.Thread `json:"~thread,omitempty"`
-	WebRedirect interface{}       `json:"~web-redirect,omitempty"`
+	WebRedirect interface{}       `json:"~web_redirect,omitempty"`
```

Comment:
This is DIDCommV1 web redirect, do not change this.

Reply:
web_redirect change reverted